src/components/form.js

```diff
@@ -43,15 +43,20 @@ class Label extends PureComponent {
     return (
       <label
         className={cx('control-label', `col-${this.props.size}`)}
+        title={this.props.title}
```

Comment:
This assumes that the string is already translated, right? Other components with tooltips assume that they are being passed an id (and translate the id themselves). With input coming from templates, vocabulary or plugins this gets tricky, because translation may have to be handled outside but we should aim for all our components to handle these props in the same way.